stream_alert/athena_partition_refresh/main.py

```diff
@@ -482,15 +512,22 @@ def handler(*_):
 
     # Initialize the SQS client and recieve messages
     stream_alert_sqs = StreamAlertSQSClient(config)
-    stream_alert_sqs.get_messages()
+    # Get the first batch of messages from SQS.  If there are no
+    # messages, this will exit early.
+    stream_alert_sqs.get_messages(max_tries=2)
 
     if not stream_alert_sqs.received_messages:
-        LOGGER.info('No messages recieved, exiting')
+        LOGGER.info('No SQS messages recieved, exiting')
         return
 
+    # If the max amount of messages was initially returned,
+    # then get the next batch of messages
+    if len(stream_alert_sqs.received_messages) == 20:
```

Comment:
Your comment seems to indicate that the max amount of messages here == 20, but [this code above](https://github.com/airbnb/streamalert/pull/275/commits/ba11a9d10af46736ea225d60c53743dfd47ec65a#diff-3159770d8fb42356c5e4163f106792c1R414) looks like it sets the default to `10`.. 

As an aside, could we set the `max_tries`, `max_values`, and `max_messages` default values as class constants that can be referenced throughout.